### DISCUSSION TITLE
自定义主题报错

### DISCUSSION THREAD

**Discussion by allen0234** [web-infra-dev/rspress#504] [Q&A]
按照文档 https://rspress.dev/zh/guide/advanced/custom-theme.html 操作，能正常运行，但是vscode里面提示找不到模块“react/jsx-runtime”或其相应的类型声明。不知道是什么情况？
![image](https://github.com/web-infra-dev/rspress/assets/49856281/f3757adf-ae62-4fe5-bba1-6abefe7633a0)

**Comment by sanyuan0704** [comment]
tsconfig 里面设置一下 "jsx": "react-jsx"

**Reply by allen0234** [reply]
这个默认已经有了。
![image](https://github.com/web-infra-dev/rspress/assets/49856281/a9876ee6-1d45-4ab6-9642-d97496e54554)

**Comment by sanyuan0704** [comment]
安装一下 react 包

**Reply by allen0234** [reply]
还是不行。但是我改成 "jsx": "preserve"之后，报错消失了。
